pyproject.toml

```diff
@@ -26,6 +26,7 @@ find-tables = "redash_toolbelt.examples.find_table_names:main"
 clone-dashboard-and-queries = "redash_toolbelt.examples.clone_dashboard_and_queries:main"
 export-queries = "redash_toolbelt.examples.query_export:main"
 redash-migrate = "redash_toolbelt.examples.migrate:main"
+delete-queries = "redash_toolbelt.examples.delete_queries:main"
```

Comment:
Does this really require an example script? If it does, "delete queries" (plural) is the wrong name for it since your example accepts a single query_id (not plural).

Not every example requires an entry under `tool.poetry.scripts` either. The purpose of examples is to show _how to use redash-toolbelt_.

I suppose it's okay to have such a simple example here, but it should be named correctly.